src/App.tsx

```diff
@@ -15,48 +16,102 @@ export const goodsFromServer = [
   'Garlic',
 ];
 
+interface FilterParems {
+  sortField: string;
+  reverseOrder: boolean;
+}
+
+function getPrepearedGoods(goods: string[],
+  { sortField, reverseOrder }: FilterParems) {
+  let prepearedGoods = [...goods];
+
+  if (sortField) {
+    prepearedGoods = prepearedGoods.sort((good1, good2) => {
+      switch (sortField) {
+        case 'Sort alphabetically':
```

Comment:
```suggestion
        case SortType.alphabet:
```
Consider creating an enum and using it